compiler/ir/backend.py/src/org/jetbrains/kotlin/ir/backend/py/transformers/irToPy/toValidPythonUtils.kt

```diff
@@ -8,6 +8,8 @@ package org.jetbrains.kotlin.ir.backend.py.transformers.irToPy
 // The below utils are needed temporarily, to make an incomplete Python output at least valid Python.
 // Thanks to this, the interpreter can run the file and we can start with implementing simpler pieces.
 
+private val fqnWithoutHex = """(visit[a-zA-Z0-9_]+_[A-Z][a-zA-Z0-9]+)(_[a-z0-9]+)?""".toRegex()
```

Comment:
Well, it can be even `a-f`, but it's minor

```suggestion
private val fqnWithoutHex = """(visit[a-zA-Z0-9_]+_[A-Z][a-zA-Z0-9]+)(_[a-f0-9]+)?""".toRegex()
```